internal/model/docker_info.go

```diff
@@ -101,17 +101,24 @@ func (i ImageTarget) MaybeFastBuildInfo() *FastBuild {
 	switch details := i.BuildDetails.(type) {
 	case FastBuild:
 		return &details
+	case StaticBuild:
+		return details.FastBuild
 	case CustomBuild:
 		return details.Fast
 	}
 	return nil
 }
 
 func (i ImageTarget) FastBuildInfo() FastBuild {
-	ret, _ := i.BuildDetails.(FastBuild)
-	return ret
+	fb := i.MaybeFastBuildInfo()
```

Comment:
i'm confused...why did this change?

after this change, FastBuildInfo() and MaybeFastBuildInfo() do the exact same thing?

as far as i can tell, FastBuildInfo() is only used in one place, and this would be the wrong behavior in that case

Reply:
...I changed it b/c matt and I were confused about WHY the behavior was different but then I did the wrong thing, let me fix.